plugins/com.google.cloud.tools.eclipse.preferences.test/src/com/google/cloud/tools/eclipse/preferences/PreferenceUtilTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.eclipse.preferences;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IWorkspace;
+import org.eclipse.core.resources.ProjectScope;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.preferences.IEclipsePreferences;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PreferenceUtilTest {
+
+  private IProject project;
+  
+  @Before
+  public void setUp() {
+    IWorkspace workspace = ResourcesPlugin.getWorkspace();
+    project = workspace.getRoot().getProject("testproject" + Math.random());
```

Comment:
We could use here `com.google.cloud.tools.eclipse.appengine.libraries.TestProject` if #771 were fixed.